fastlane/Fastfile

```diff
@@ -7,8 +7,16 @@ unless FastlaneCore::Helper.bundler?
   UI.user_error!('Please run fastlane via `bundle exec`')
 end
 
+# Paths that are re-used across multiple lanes
+PROJECT_ROOT_FOLDER=File.dirname Dir.pwd
```

Comment:
Not sure if it's safe to base the logic on `pwd` – i.e. the current directory from where the user runs the script. What if the user runs `fastlane` from the root dir vs from inside the `fastlane/` folder?

I think that given how fastlane runs its Fastfile, in some way it guarantees that the `pwd ` is always inside the `fastlane/` subfolder when executing the `Fastfile`, but still, feels a bit brittle compared to basing that on the `File.dirname(File.dirname(__FILE__))` or something like that? (needs some testing tho, didn't check that `__FILE__` would work in fastlane's context)

Reply:
I actually went back and forth on this – I originally had it the other way, but figure this was more idiomatic 😅 

Changed in 32241c00ae722b6cfce096c3e298a6b53d44af87